File: fixture/C/input/Block1.c
#include <stdio.h>

int main() {
	int i = 0;
	
	if (i == 0)
		printf("test");
	
	if (i <= 0 && (i >= 0 || i != 0))
		printf("test");
	
	switch(i)
		case 0:
		printf("test");
	
	while (i != 0)
		printf("test");
	
	do
		printf("test");
	while (i != 0);
	
	for (i = 0; i < 0; i++)
		printf("test");
	return 0;
}

File: fixture/JavaScript/input/Block1.js
function func() {
	i = 0;
	
	if (i == 0)
		alert("test");
	
	switch(i) {
		case 0:
		alert("test");
	}
	
	while (i != 0)
		alert("test");
	
	do
		alert("test");
	while (i != 0);
	
	for (i = 0; i < 0; i++)
		alert("test");

	for (i in new Array())
		alert("test");
	
	obj = { a: 1 }
	with(obj)
		alert(a)

	return 0;
}
